packages/vuetify/src/labs/VTreeview/VTreeview.tsx

```diff
@@ -146,7 +147,8 @@ export const VTreeview = genericComponent<new <T>(
     })
 
     useRender(() => {
-      const listProps = VList.filterProps(props)
+      const listProps = VList.filterProps(vm.vnode.props!)
```

Comment:
Avoid non-existing prop being assigned as `undefined` to VList

If `@update:opened="openedListener"` is used in Treeview, `VList.filterProps(props)` is equivalent to `<VList :opened="undefined" @update:opened="openedListener">`, which breaks VList [demo](https://play.vuetifyjs.com/playgrounds/Ba6uEA)